### ISSUE TITLE
Convert to use `horde_sdk`

### ISSUE THREAD

**Issue by tazlin** [Haidra-Org/AI-Horde-CLI#13]
The horde_sdk naturally leads into needing a reference implementation of a client; the cli would be an ideal ground for this.

- It is hard to gauge how large the audience is, but ideally it should be implemented so the running the scripts behave the same way as the current version, so users' workflows are not broken on update.

**Comment by db0** [comment]
Agreed. This was my expected first use as the CLI was always the "PoC" for the AI Horde for integrators to see
